apps/web/components/apps/installation/EventTypesStepCard.tsx

```diff
@@ -4,6 +4,7 @@ import React from "react";
 import { useFieldArray, useFormContext } from "react-hook-form";
 
 import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { markdownToSafeHTML } from "@calcom/lib/markdownToSafeHTML";
```

Comment:
Don't we need to use markdownToSafeHTMLClient here? Could you please verify? Apart from this, it's okay